app/javascript/hyacinth_v1/components/digital_objects/metadata/MetadataForm.js

```diff
@@ -59,9 +60,13 @@ function MetadataForm(props) {
   } = digitalObject;
   const [descriptiveMetadata, setDescriptiveMetadata] = useState({});
   const [createDigitalObject, { error: createErrors }] = useMutation(createDigitalObjectMutation);
```

Comment:
@JackBlackLight Looking at this again, I realized that we'll also want to handle userErrors that come from the createDigitalObject mutation.  Since we don't create AND update in the same operation, we can safely merge both create and update user errors into the same array (since one set will always be blank).